src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -854,7 +854,7 @@ VideoDecoder::AudioFramesOutput VideoDecoder::getFramesPlayedInRangeAudio(
 
   if (startSeconds == stopSeconds) {
     // For consistency with video
-    return AudioFramesOutput{torch::empty({0}), 0.0};
+    return AudioFramesOutput{torch::empty({0, 0}), 0.0};
```

Comment:
Drive-by, the video APIs return something of shape `(0, C, H, W)` (where C H W are from the metadata). Here, (0, 0) is the best we can do, at least it preserves the number of dimensions.